blackbox/docs/enterprise/monitoring.txt

```diff
@@ -0,0 +1,106 @@
+.. _jmx_monitoring_plugin:
+
+=====================
+JMX Monitoring Plugin
+=====================
+
+The CrateDB monitoring plugin exposes query metrics via the `JMX`_ API.
+
+This plugin is a component of the `Enterprise Edition`_ of CrateDB.
+
+To use this plugin, you need to:
+
+- Set ``license.enterprise`` in the CrateDB settings
+- Enable stats
+- Enable the JMX API
+
+Enable Stats
+------------
+
+.. highlight:: psql
+
+To monitor query metrics, the ``stats.enabled`` cluster setting must be enabled::
+
+  cr> SET GLOBAL "stats.enabled" = TRUE;
+
+Enabling the JMX API
+--------------------
+
+.. highlight:: none
+
+To monitor CrateDB using the JMX API, you must set the following system properties before you start CrateDB::
+
+  com.sun.management.jmxremote
+  com.sun.management.jmxremote.port=<JMX_SERVER_PORT>
+  com.sun.management.jmxremote.ssl=false
+  com.sun.management.jmxremote.authenticate=false
+
+Here, ``<JMX_SERVER_PORT>`` should be replaced with the port number of your JMX server.
```

Comment:
regarding ssl and authenticate, we can say that we have to set it to false, such as we do not support JMX password authentication and SSL